src/Microsoft.AspNetCore.Mvc.Razor/TagHelperComponent/BodyHeadTagHelper.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.Mvc.Razor.TagHelperComponent
+{
+    /// <summary>
+    /// A <see cref="TagHelperComponentTagHelper"/> targeting &lt;body&gt; and &lt;head&gt; elements.
+    /// </summary>
+    [HtmlTargetElement("body")]
+    [HtmlTargetElement("head")]
+    public class BodyHeadTagHelper : TagHelperComponentTagHelper
```

Comment:
cc @DamianEdwards about that... unless something changed, I thought we wanted just one. Not that I guess there's much/any downside to having two tag helpers.